src/backend/linalg_internal_cpu/Add_internal.hpp

```diff
@@ -2,754 +2,83 @@
 #define CYTNX_BACKEND_LINALG_INTERNAL_CPU_ADD_INTERNAL_H_
 
 #include <assert.h>
-#include <iostream>
-#include <iomanip>
 #include <vector>
 #include "backend/Storage.hpp"
 #include "Type.hpp"
+#include "utils/utils.hpp"
+
+namespace {
```

Comment:
The style guide doesn't allow using unnamed namespace in the header files. See: https://google.github.io/styleguide/cppguide.html#Internal_Linkage

I think the reason is that this will cause every transition unit own its copy of the namespace and make problem when applying [Include What You Use](https://google.github.io/styleguide/cppguide.html#Include_What_You_Use) rule.

Can the implementation of `AddInternalCore` be put in `AddInternalImpl` function directly?